service/controller/v4/version_bundle.go

```diff
@@ -31,7 +31,7 @@ func VersionBundle() versionbundle.Bundle {
 		Components: []versionbundle.Component{
 			{
 				Name:    "calico",
-				Version: "3.0.8",
+				Version: "3.2.0",
```

Comment:
Align with reality

Reply:
3.2.3?

Reply:
2.0.0 bundle has 3.2.0 calico, but we forgot to update this.